web_src/less/themes/arc-green.less

```diff
@@ -1,70 +1,73 @@
 @import "_base";
 
-.hljs {
-    display: block;
-    overflow-x: auto;
-    padding: 0.5em;
-    color: #bababa;
-}
+/* the html parent is needed for specificity against lazy-loaded themes */
+html {
```

Comment:
Should this be labeled as breaking, for users that have their own styles? (I don't know our policy there)

Reply:
It should not be breaking as long as the user has set the default theme. This is the first time I hear user styles exists. Generally I want to rework themeing (and coloring) to use CSS variables, which will make adding custom themes a lot easier than it currently is.

Reply:
Related #6011

Reply:
That theme does not modify syntax, but I guess I will just include `highlight.js/styles/github.css` in the main CSS bundle so it will load like before. It's less than 1KB anyways.